website/docs/reference/integrations/jira-server-plugin-installation.md

```diff
@@ -1,88 +0,0 @@
----
```

Comment:
I think we should mark this as deprecated rather than just remove it.

Lots of folks are stuck on the old Jira version (software moves sloooow). So we shouldn't remove helpful docs for them, just warn them that they should try to get their stuff upgraded, what do you think?